src/einsteinpy/tests/test_symbolic/test_predefined/test_find.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+from einsteinpy.symbolic.predefined import find
+
+
+def test_find_correct():
+    assert find("sit") == ["AntiDeSitter", "AntiDeSitterStatic", "DeSitter"]
```

Comment:
The order of output may change once more metrics are added in `predefined`.
Therefore it would be good to test if one is subset of other

```
names = ["AntiDeSitter", "AntiDeSitterStatic", "DeSitter"]
find_list = find("sit")
assert set(names).issubset(set(find_list))
```